src/JavaBasic.java

```diff
@@ -1,5 +1,103 @@
+import static java.lang.String.*;
 public class JavaBasic {
-    public static void main(String[] args){
-        
+    public static void remove() {      // this method removes letter s from string
+        String sample = "This is very easy programme"; // sample string
+        sample = sample.replace( "s", "" );
+        System.out.print( "1) String without s will look like = " ); //// result description for printing sample length
+        System.out.println( sample );  // print result for string sample
+    }
+    public static void findlength() { // this method created to find length of string
+        String sample = "This is very easy programme"; // sample string
+        System.out.print( "2) Lenght of string is = " ); // result description for printing sample length
+        System.out.println( sample.length() ); //prints results for length of string
+    }
+
+    public static void removespace() { // method for removing space in string
+        String sample = "This is very easy programme"; // sample string
+        System.out.print( "3) String without space will look like = " ); // result description for printing string
+        System.out.println( sample.replaceAll( "\\s", "" ) ); // print result string without space
+    }
+
+    public static void extractspechar() { // method for extracting special characters from string
+        String sample = "This $300 is equivalent of £200 but not for @100"; // sample string with special characters
+        System.out.print( "4) Removing special Characters = " ); // result description for printing string
+        System.out.println( sample.replaceAll( ("[^a-zA-Z0-9]"), " " ) );
```

Comment:
```suggestion
        System.out.println( sample.replaceAll( ("[^a-zA-Z0-9]"), "" /* no space */ ) );
```
with your code you are replacing special characters by space, but you need to remove it so replace by empty string ""  , good that you used regex

Reply:
i have changed From " " to ""
and also changed method names as you suggested with camel case. i will create push it again now